UDV-Core/examples/MAM/MAM.js

```diff
@@ -0,0 +1,294 @@
+var help  = new udvcore.HelpWindow({active:true});
+
+// An html container is required in order to control the placement of the
+// dat.GUI object within the page.
+var datDotGUIDiv = document.createElement("div");
+datDotGUIDiv.id = 'datDotGUIDiv';
+document.body.appendChild(datDotGUIDiv);
+
+// Associate the stylesheet for layout configuration
+var link = document.createElement('link');
+link.setAttribute('rel', 'stylesheet');
+link.setAttribute('type', 'text/css');
+link.setAttribute('href', './MAM.css');
+document.getElementsByTagName('head')[0].appendChild(link);
+
+// Proceed with the creation of the dat.GUI with the above positionning
+var datDotGUI = new dat.GUI({ autoPlace: false });
+datDotGUI.domElement.id = 'datDotGUI';
+var datDotGUIContainer = document.getElementById('datDotGUIDiv');
+datDotGUIContainer.appendChild( datDotGUI.domElement );
+
+// About subwindow
+aboutController = datDotGUI.add( about, 'windowIsActive'
+                               ).name( "About" ).listen();
+aboutController.onFinishChange( function(value) { about.refresh(); } );
+
+// About subwindow
+helpController = datDotGUI.add( help, 'windowIsActive'
+                              ).name( "Help" ).listen();
+helpController.onFinishChange( function(value) { help.refresh(); });
+
+//datDotGUI.close();     // By default the dat.GUI controls are rolled up
+
+
+for (const layer of view.getLayers()) {
+	layer.whenReady.then( function _(layer) {
+		var gui = debug.GeometryDebug.createGeometryDebugUI(datDotGUI, view, layer);
+		debug.GeometryDebug.addMaterialLineWidth(gui, view, layer, 1, 10);
+	})
+}
+
+// Keyboard Controller
+document.addEventListener('keydown', (event) => {
+  if (event.key === '1') {
+	  //Switch the 3D building Layer visibility
+	  for (const layer of view.getLayers()) {
+		  if (layer.id === 'WFS Bus Lines') {
+			console.log(event.key);
+			layer.visible = !layer.visible;
+			//Request redraw
+			view.notifyChange();
+		}
+	}
+    return;
+  }
+
+  if (event.key === '2') {
+	  //Switch the BusLine Layer visibility
+	  for (const layer of view.getLayers()) {
+		  if (layer.id === 'WFS Buildings') {
+			  console.log(event.key);
+			  layer.visible = !layer.visible;
+			//Request redraw
+			view.notifyChange();
+		}
+	}
+    return;
+  }
+
+  if (event.key === '3') {
+	  //Switch the BusLine Layer visibility
+	  for (const layer of view.getLayers()) {
+		  if (layer.id === 'WMS Pollution Air') {
+			  console.log(event.key);
+			  layer.visible = !layer.visible;
+			//Request redraw
+			view.notifyChange();
+		}
+	}
+    return;
+  }
+
+  if (event.key === 'a') {
+	  //Advanced controller (etiding option)
+    if (confirm('Do you want to switch controller option ?\n Current Option '+((useControlsForEditing)? "Edit Setting" : "Regular Setting"))) {
+		useControlsForEditing = !useControlsForEditing; //Change Option
+    alert((useControlsForEditing)? "Edit Setting Activate" : "Regular Setting Activate");//Inform about new setting
+    } else {
```

Comment:
If nothing is done in the else condition you can remove it.

Reply:
Nothing happened in else condition (the else condition is removed).